tests/integration_tests/charts/api_tests.py

```diff
@@ -1076,6 +1076,55 @@ def test_get_chart_not_found(self):
         rv = self.get_assert_metric(uri, "get")
         assert rv.status_code == 404
 
+    def test_slice_get_by_id(self):
```

Comment:
Could we merge these tests together and avoid duplication? both for the success and non existent ones

Reply:
Yes, I mean, looking at the tests you can assert both things just with 1 extra assertion that's why I suggested merging them, but the ultimate goal is to DRY them because those tests are 95% the same